components/static/Logo.vue

```diff
@@ -66,6 +66,8 @@ defineProps(['class'])
 #logo {
   width: 22rem;
   height: 22rem;
+  cursor: pointer;
+  // ^ added by misterk7_-. todo: disable when logo isn't interactable.
```

Comment:
There is no way to tell when the logo is interactive without adding an extra class during the input stage
```diff
- cursor: pointer;
- // ^ added by misterk7_-. todo: disable when logo isn't interactable.
+ // imply interactivity through cursor when acting as an input
+ &.interactive { cursor: pointer; }
```
While this is a good idea, I would've just sent this as a github issue or a suggestion in `#packxbr-suggestions`, because:
1. you didn't have the full solution
2. its very small (+2 lines excluding white space and comments)